webroot/adm/client-key.html

```diff
@@ -33,7 +33,6 @@ <h3>Operations</h3>
     <li class="ro-cki" style="display: none"><a href="#" id="doEnable">Enable Client Key</a></li>
     <li class="ro-cki" style="display: none"><a href="#" id="doSetRoles">Set Roles</a></li>
     <li class="ro-cki" style="display: none"><a href="#" id="doSetSite">Update Site</a></li>
-    <li class="ro-adm" style="display: none"><a href="#" id="doRekey">Rekey</a></li>
     <li class="ro-adm" style="display: none"><a href="#" id="doDel">Delete Client Key</a></li>
```

Comment:
Can you edit the HTML to add a heading saying something like "Low Level Operations - Do Not Use Unless Necessary" for Delete and Rename? This will help resolve one of the IM action items from a recent IM